pkg/plugins/pfs/corelist/corelist_load_gen.go

```diff
@@ -13,9 +13,10 @@ import (
 	"fmt"
 	"io/fs"
 
+	"github.com/grafana/thema"
+
```

Comment:
Running `make gen-go` will revert the changes in this file because it's auto generated:

```
➜  grafana git:(zserge/goimports-plugins-platform) ✗ make gen-go
generate code from .cue files
go generate ./pkg/plugins/plugindef
go generate ./kinds/gen.go
go generate ./public/app/plugins/gen.go
go generate ./pkg/kindsys/report.go
generate go files
/Users/nyo/go/bin/wire-v0.5.0 gen -tags "oss" ./pkg/server ./pkg/cmd/grafana-cli/runner
wire: github.com/grafana/grafana/pkg/server: wrote /Users/nyo/grafana/grafana/pkg/server/wire_gen.go
wire: github.com/grafana/grafana/pkg/cmd/grafana-cli/runner: wrote /Users/nyo/grafana/grafana/pkg/cmd/grafana-cli/runner/wire_gen.go
➜  grafana git:(zserge/goimports-plugins-platform) ✗ git diff | cat
diff --git a/pkg/plugins/pfs/corelist/corelist_load_gen.go b/pkg/plugins/pfs/corelist/corelist_load_gen.go
index 0ede6eae4f..4ba87ce393 100644
--- a/pkg/plugins/pfs/corelist/corelist_load_gen.go
+++ b/pkg/plugins/pfs/corelist/corelist_load_gen.go
@@ -13,10 +13,9 @@ import (
 	"fmt"
 	"io/fs"

-	"github.com/grafana/thema"
-
 	"github.com/grafana/grafana"
 	"github.com/grafana/grafana/pkg/plugins/pfs"
+	"github.com/grafana/thema"
 )

 func parsePluginOrPanic(path string, pkgname string, rt *thema.Runtime) pfs.ParsedPlugin {
```

I would exclude this file from the changes or change the generator

Reply:
Yeah, I agree it's probably good idea to skip code-generated files.